bluebrain/documentation/setup_personal.md

```diff
@@ -45,31 +52,29 @@ by using Spack's environment feature.
 
 ## Building software on Ubuntu
 
-Use Spack on the workstations provided by the project.
+Ubuntu / Debian have a habit of being somewhat _special_, patching upstream
+projects in unexpected ways.
+Consider yourself warned to not rely on system packages and defer to
+Spack-installed ones when needed.
 
-### Ubuntu 18.04
+First, ensure that the essential packages to building stuff are installed:
 
-We build Docker images based on Ubuntu 18.04, and the same settings can be
-used to set Spack up on the desktops:
-
-    $ git clone -c feature.manyFiles=true https://github.com/BlueBrain/spack.git
-    $ mkdir ~/.spack
-    $ cp spack/bluebrain/sysconfig/ubuntu-18.04/*.yaml ~/.spack
-    $ sed -e 's/#.*//g' spack/bluebrain/sysconfig/ubuntu-18.04/packages|xargs -r sudo apt-get install --assume-yes
-    $ . spack/share/spack/setup-env.sh
-    $ spack compiler find
+    $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    $ sudo apt update
+    $ sudo apt install build-essential gcc-11 g++-11 gfortran-11
 
-### Ubuntu 20.04
+Then clone and configure Spack:
 
     $ git clone -c feature.manyFiles=true https://github.com/BlueBrain/spack.git
```

Comment:
```suggestion
    $ git clone -c feature.manyFiles=true git@github.com:BlueBrain/spack.git
```

Reply:
In this way it does not complain about keys...

Reply:
I don't know which keys but this should be `https` as repo is public.

Reply:
I am talking about the fact that github stopped accepting simple name/password and requires a key to commit

Reply:
can't you use git:*** if the repo is public?

Reply:
I just use `gh repo clone BlueBrain/spack`, and it sets up `https` by default, IIRC